tests/storage/hotplug.go

```diff
@@ -1260,10 +1260,11 @@ var _ = SIGDescribe("Hotplug", func() {
 				}
 			})
 
-			DescribeTable("should be able to add and remove volumes", func(addVolumeFunc addVolumeFunction, removeVolumeFunc removeVolumeFunction, storageClassFunc storageClassFunction, volumeMode k8sv1.PersistentVolumeMode, vmiOnly bool) {
-				sc, exists := storageClassFunc()
+			DescribeTable("should be able to add and remove volumes", decorators.RequiresBlockStorage, func(addVolumeFunc addVolumeFunction, removeVolumeFunc removeVolumeFunction, volumeMode k8sv1.PersistentVolumeMode, vmiOnly bool) {
```

Comment:
not all the tests here require the block storage, we probably shouldnt skip the FS once in lanes without block storage

Reply:
check the comment a line below;
they all want filesystems on top of a block device (I could not figure out the reason)
but I prefer to keep the state equivalent to prior to this PR

Reply:
Oh sorry I missed the fact that all the tests passed the check of the blockSC exists, which is weird since if so why did they even send this variable but cool